communities/src/lib.rs

```diff
@@ -67,58 +75,77 @@ decl_module! {
         // where n is the number of all locations of all communities
         // this should be run off-chain in substraTEE-worker later
         #[weight = 10_000]
-        pub fn new_community(origin, loc: Vec<Location>, bootstrappers: Vec<T::AccountId>) -> DispatchResult {
+        pub fn new_community(
+            origin,
+            loc: Vec<Location>,
+            bootstrappers: Vec<T::AccountId>,
```

Comment:
As you're touching the code here, I'd suggest to resolve #14 at the same time.
Every `vec` argument should be constrained in size, i.e. with `ensure` at the beginning of the fn

Locations: no more than 1000 for now (because of #9)
bootstrappers: no more than 12